### PR TITLE
show stdout on xcode failures

### PR DESCRIPTION
**Pull request by devoncarew**
Show stdout on xcode failures. Xcode passes useful info on failures to both stderr and stdout.

@chinmaygarde

### CODE REVIEW

File: packages/flutter_tools/lib/src/ios/mac.dart [hunk 1]
```diff
@@ -122,12 +122,20 @@ Future<bool> buildIOSXcodeProject(ApplicationPackage app, { bool buildForDevice
     commands.addAll(<String>['-sdk', 'iphonesimulator', '-arch', 'x86_64']);
   }
 
-  try {
-    runCheckedSync(commands, workingDirectory: app.localPath);
-    return true;
-  } catch (error) {
-    return false;
+  printTrace(commands.join(' '));
```
Comment (yjbanov) [inline packages/flutter_tools/lib/src/ios/mac.dart]: perhaps add a human-readable description of what this trace is about?

**Comment by yjbanov** [comment]
lgtm

File: packages/flutter_tools/lib/src/ios/mac.dart (hunk 1 above)
Comment (devoncarew) [inline packages/flutter_tools/lib/src/ios/mac.dart]: It'll end up being pretty readable:

```
/usr/bin/env xcrun xcodebuild -target Runner -configuration Release -sdk iphoneos -arch arm64
```

Comment (yjbanov) [inline packages/flutter_tools/lib/src/ios/mac.dart]: Sounds good then.

**Pull request merged**